docs/zz_generated.kubermaticConfiguration.yaml

```diff
@@ -467,20 +467,21 @@ spec:
         - from: 1.21.*
           to: 1.21.*
         - automatic: true
-          from: '>= 1.21.0, < 1.21.7'
-          to: 1.21.7
+          from: '>= 1.21.0, < 1.21.8'
+          to: 1.21.8
         - from: 1.21.*
           to: 1.22.*
         - from: 1.22.*
           to: 1.22.*
         - automatic: true
-          from: '>= 1.22.0, < 1.22.4'
-          to: 1.22.4
+          from: '>= 1.22.0, < 1.22.5'
+          to: 1.22.5
       # Versions lists the available versions.
       versions:
         - 1.20.13
```

Comment:
Why are we not doing automatic upgrades `1.20.13` -> `1.20.14` as with the other releases?

Reply:
I don't think it makes much sense. 1.20.14 is mostly a bug fix release, it doesn't include CVE fixes, so I don't think an automatic upgrade is mandatory. However, other releases include CVE fixes, so it makes sense to auto-upgrade.

Reply:
Aha! I wasn't aware that 1.20.14 would not include a CVE fix like the 1.21.x and 1.22.x releases. Thanks.

Reply:
For reference, that's because 1.20 uses Go 1.15, which is not maintained by the Go team any longer.